### DISCUSSION TITLE
Wildcard or dynamic selfservice.whitelisted_return_urls

### DISCUSSION THREAD

**Discussion by davidthor** [ory/kratos#993] [Q&A] [answered]
I'm trying to setup Kratos as the user management system for my org, but am trying to do so in such a way that multiple first-party web apps could share this system. All webapps would be deployed and hosted using the same base domain (e.g. `*.example.com`), but I don't want to have to restrict the subdomains themselves. For the most part, I can configure Kratos to be completely independent of these upstream consumers except for with `selfservice.whitelisted_return_urls`, which doesn't seem to respond to wildcard usage or have any way to be configured dynamically.

Is there any way to dynamically set valid return_to URLs or use wildcards to support multiple subdomains without explicit registration? I don't want to have to reconfigure and restart the user management system every time a new first-party app wants to leverage the user management system. 

Thanks in advance.

**Comment by zepatrik** [comment, marked as answer]
This is already tracked as #943 :wink: 
Please head over to the discussion there. Kratos supports hot reloading the config, so it should be possible to dynamically add a organization while the service is up (as a workaround for now).

**Reply by davidthor** [reply]
I can't believe i missed that. Thanks for pointing it out! I'll continue the thoughts there.
